CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,3 @@
+- Use AMQP durable option in assertExchange
+- Use device apikey if exists in getEffectiveApiKey for command handling
+- Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life 
```

Comment:
I think the conflict with master wasn't correctly solved... Note that CHANGES_NEXT_RELEASE in mater  (https://github.com/telefonicaid/iotagent-ul/blob/master/CHANGES_NEXT_RELEASE) is empty, so this PR should only include L3.

L1-L2 where included in last release (https://github.com/telefonicaid/iotagent-ul/releases/tag/1.12.0)

Reply:
Fixed 33cbd86.